### DISCUSSION TITLE
Is Supabase Realtime cheap to scale?

### DISCUSSION THREAD

**Discussion by hmmhmmhm** [supabase/supabase#14597] [Questions]
Supabase has a realtime feature, which I understand starts with a limit of 500 users in the enterprise. However, this is not serverless and I doubt I can pick and choose the performance I want to use when scaling horizontally.
Is it advisable to use Supabase Realtime and Broadcast instead of using a serverless memcached database like Upstash? Isn't this only scaling to expensive equipment?
It seems that the performance of PostgreSQL DB instance and Supabase Realtime instance should change at the same time as the scale increases, but isn't scaling expensive and has many limitations?

**Comment by ArmanAryanpour** [comment]
I believe it is 500 users in pro and unlimited in Enterprise as per the pricing page: https://supabase.com/pricing

In terms of scaling, I believe it all depends on how time sensitive you are in your application. 

And I would assume on the supa's end, it is an aggregated and auto-scalable infrastructure although not server less.

Supabase Realtime has been pretty reliable for us.

**Reply by hmmhmmhm** [reply]
Liveblock states how much may have to pay per MAU, I think supabase should have such a plan page as well. Supabase has the same Presence and Broadcast features as Liveblock. and LiveBlock costs a whopping $939 per month for 10000 MAUs. 

Whenever I introduce Supabase to developers around me, I hear a lot of concerns about the cost. I need some resources to introduce it to them. I know that Supabase is a great SaaS that is also available on-premises. But for potential customers looking for Enterprise SaaS, this is not an advantage.

**Reply by ArmanAryanpour** [reply]
I agree about the transparent pricing. 

I do also trust that supabase is not purely motivated by money and they currently offer amazing features for the price charged and historically have done the same. A similar setup on the cheapest cloud provider even for the pro or free tier with this quality will not cost less than what Supabase charges. 

Liveblock is very expensive, at least 7x the price even without enterprise discount. 

Liveblock 10000MAU: $939
Supabase 10500MAU: 500MAU @ $25 +( ($10 x 1000MAU) x 10 = $100 ) = $125

I think it is safe to bet that supabase will be cheaper and you still own the code, data, and can always go somewhere else for the server.

**Comment by hmmhmmhm** [comment]
I know that the Supabase team recently revamped the spend portion of the bill that is visible to enterprise users.
I was wondering if the team could also revamp the pricing page that is visible to potential customers (practitioners in large organizations or startups)?

For example, a Docker hosting service called Railway has a slider that they offer consumers to approximate the cost. I think Supabase could have more customers if they shared a more transparent SaaS pricing policy.

[Railway Pricing](https://railway.app/pricing)
![6346342](https://github.com/supabase/supabase/assets/38493722/75f30c7f-8c0d-4d98-801f-b90fde595ab7)
